### PR TITLE
story: Update endpoint for download links CAVL

### PR DESCRIPTION
**Pull request by alainajaffer**

### CODE REVIEW

File: transit_odp/avl/tasks.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def task_validate_avl_feed(task_id: str):
 
 @shared_task(bind=True)
 def task_create_sirivm_zipfile(self):
-    URL = f"{settings.CAVL_CONSUMER_URL}/datafeed"
+    URL = f"{settings.CAVL_API_BASE_URL}/siri-vm"
```
Comment (nbetts) [inline transit_odp/avl/tasks.py:119]: Do these changes need a feature flag, in case bods does a release before our service is ready?

Comment (nbetts) [reply transit_odp/avl/tasks.py:119]: by changes I mean this line and the changed line further down

**Comment by nbetts** [comment]
The ticket also mentions updating the `/datafeed/{datafeedID}` endpoint, which in our system will be `/siri-vm?subscriptionId={datafeedID}`

**Comment by alainajaffer** [comment]
> The ticket also mentions updating the `/datafeed/{datafeedID}` endpoint, which in our system will be `/siri-vm?subscriptionId={datafeedID}`

https://github.com/department-for-transport-BODS/bods/pull/1219/files updated in this which the branch is off

**Review by nbetts — APPROVED** [review]

**Pull request merged**